docs/index.txt

```diff
@@ -159,6 +159,8 @@ CrateDB JDBC driver supports following properties:
     If this property is set to ``true`` the driver will be compliant with the JDBC
     specification. It will then raise an unsupported operation exception if an
     unsupported operation is called , like auto-commit or setting a savepoint.
+    It will also report that it doesn't support transactions and the only transaction isolation
```

Comment:
the para below about the default is now a bit confusing. it would be appropriate to elaborate a bit more about the behaviour of strict here.